static/_mobile.scss

```diff
@@ -57,6 +57,15 @@
   }
 }
 
+@media screen and (max-width: 899px) {
```

Comment:
@Unesdala please make this 700px, thanks

Reply:
@Unesdala i guess you then need two of these, one for 700 and above, and one for up to 699 that displays the cell version?